app/models/daos/OAuth1InfoDAO.scala

```diff
@@ -3,18 +3,38 @@ package models.daos
 import com.mohiva.play.silhouette.api.LoginInfo
 import com.mohiva.play.silhouette.impl.daos.DelegableAuthInfoDAO
 import com.mohiva.play.silhouette.impl.providers.OAuth1Info
-import models.daos.OAuth1InfoDAO._
+import javax.inject.Inject
 import play.api.libs.concurrent.Execution.Implicits._
-
-import scala.collection.mutable
+import play.api.db.slick.DatabaseConfigProvider
 import scala.concurrent.Future
 
 /**
  * The DAO to store the OAuth1 information.
- *
- * Note: Not thread safe, demo only.
  */
-class OAuth1InfoDAO extends DelegableAuthInfoDAO[OAuth1Info] {
+class OAuth1InfoDAO @Inject() (protected val dbConfigProvider: DatabaseConfigProvider)
```

Comment:
SilhouetteModule expects implementations for OAuth. Removing these classes will lead to the following compile time error. We could make this behaviour explicit by adding a comment or replacing the OAuth-DAOs with mock-implementations.

```
CreationException: Unable to create injector, see the following errors:

1) No implementation for com.mohiva.play.silhouette.impl.daos.DelegableAuthInfoDAO<com.mohiva.play.silhouette.impl.providers.OAuth1Info> was bound.
  at modules.SilhouetteModule.provideAuthInfoRepository(SilhouetteModule.scala:115) (via modules: com.google.inject.util.Modules$OverrideModule -> modules.SilhouetteModule)

2) No implementation for com.mohiva.play.silhouette.impl.daos.DelegableAuthInfoDAO<com.mohiva.play.silhouette.impl.providers.OAuth2Info> was bound.
  at modules.SilhouetteModule.provideAuthInfoRepository(SilhouetteModule.scala:115) (via modules: com.google.inject.util.Modules$OverrideModule -> modules.SilhouetteModule)
```